docker/release.sh

```diff
@@ -1,73 +1,68 @@
 #!/bin/bash
 set -ex
 
-vt_base_version='v7.0.2'
-orchestrator_version='3.2.3'
-pmm_client_version='1.17.4'
+vt_base_version='v12.0.3'
+debian_versions='buster  bullseye'
+default_debian_version='buster'
 
 docker pull vitess/base:$vt_base_version
 
-docker build --build-arg VT_BASE_VER=$vt_base_version -t vitess/k8s:$vt_base_version-buster .
-docker tag vitess/k8s:$vt_base_version-buster vitess/k8s:$vt_base_version
-docker push vitess/k8s:$vt_base_version-buster
-docker push vitess/k8s:$vt_base_version
+for debian_version in $debian_versions
+do
+  echo "####### Building vitess/vt:$debian_version"
 
-docker build --build-arg VT_BASE_VER=$vt_base_version -t vitess/vtgate:$vt_base_version-buster vtgate
-docker tag vitess/vtgate:$vt_base_version-buster vitess/vtgate:$vt_base_version
-docker push vitess/vtgate:$vt_base_version-buster
-docker push vitess/vtgate:$vt_base_version
+  docker build --build-arg VT_BASE_VER=$vt_base_version --build-arg DEBIAN_VER=$debian_version-slim -t vitess/k8s:$vt_base_version-$debian_version .
+  docker tag vitess/k8s:$vt_base_version-$debian_version vitess/k8s:$vt_base_version
+  docker push vitess/k8s:$vt_base_version-$debian_version
+  if [ $debian_version = $default_debian_version ]; then docker push vitess/k8s:$vt_base_version; fi
```

Comment:
I think that we should do this in one of two ways:
```
 if [ "$debian_version" = "$default_debian_version" ];
```
```
 if [[ $debian_version == $default_debian_version ]];
```

Single bracket is posix compliant, but requires quoting values for string comparisons. Double bracket allows for more natural operations. Double == is more explicit IMO as it's clearly full equality comparison rather than assignment, but either works with double brackets.

Reply:
I went with #2 - I trust your opinion :)